src/javascript/app/base/header.js

```diff
@@ -493,9 +494,15 @@ const Header = (() => {
 
                     const account           = createElement('div', { class: `account__switcher-acc ${is_current ? 'account__switcher-acc--active' : ''}`, 'data-value': loginid });
                     const account_icon      = createElement('img', { src: icon });
-                    const account_detail    = createElement('span', { text: is_real ? currency : 'Demo' });
+                    const account_detail    = createElement('span', { text: is_real ? (currency || localize('Real')) : localize('Demo') });
                     const account_loginid   = createElement('div', { class: 'account__switcher-loginid', text: loginid });
                     const account_balance   = createElement('span', { class: `account__switcher-balance account__switcher-balance-${is_real ? currency : 'virtual'}` });
+                    
+                    if (!currency) {
+                        $('#header__acc-balance').html(createElement('p', { text: localize('No Currency Assigned') }));
```

Comment:
Should this be in sentence case? e.g.

```suggestion
                        $('#header__acc-balance').html(createElement('p', { text: localize('No currency assigned') }));
```